hail/python/test/hail/stats/test_linear_mixed_model.py

```diff
@@ -370,3 +370,26 @@ def test_linear_mixed_regression_low_rank(self):
                                              model)
         assert np.allclose(ht.beta.collect(), beta_hail)
         assert np.allclose(ht.p_value.collect(), pval_hail)
+
+    def test_linear_mixed_regression_pass_through(self):
+        x_table = hl.import_table(resource('fastlmmCov.txt'), no_header=True, impute=True).key_by('f1')
```

Comment:
this is an expensive test. is it unreasonable to add this check to the full_rank or low_rank one?

Reply:
that does violate our testing strategy (small, independent tests)

Reply:
why is the LMM test so slow? the data isn't big.

Reply:
ok, it's not that expensive. but you create a negligible dataset for this test, or at least set log_gamma in fit rather than fitting it.

Reply:
sorry, what change do you want me to make?